### PR TITLE
[Part-1]: New analytics events

### PR DESCRIPTION
**Pull request by kristina-fefelova**
**Pull Request Requirements:**

* Provide a brief description of the changeset.
* Include a screenshots if applicable
* Ensure that the changeset adheres to the [DCO guidelines](https://github.com/apps/dco).

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NmI5YjEyMTEwMDhmZDk4YzJkZTI4MTkiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.-h70zvQBT6kXjsf5XIRwJkzaqMB2JJP7YRaxHWWnfeg">Huly&reg;: <b>UBERF-7870</b></a></sub>

### CODE REVIEW

File: plugins/contact-resources/src/components/CreateOrganization.svelte [hunk 1]
```diff
@@ -66,7 +67,7 @@
     if (onCreate !== undefined) {
       await onCreate?.(id, client)
     }
-
+    Analytics.handleEvent(ContactEvents.CompanyCreated, { id })
```
Comment (haiodo) [inline plugins/contact-resources/src/components/CreateOrganization.svelte:70]: why not have like trigger some contribution based on object kind?

File: plugins/chunter-resources/src/components/chat-message/ChatMessageInput.svelte [hunk 2]
```diff
@@ -108,7 +109,11 @@
       const res = await createMessage(event, _id, `chunter.create.${_class} ${object._class}`)
 
       console.log(`create.${_class} measure`, res.serverTime, res.time)
+      const objectId = await getObjectId(object, client.getHierarchy())
+      Analytics.handleEvent(ChunterEvents.MessageCreated, { ok: res.result, objectId, objectClass: object._class })
     } catch (err: any) {
+      const objectId = await getObjectId(object, client.getHierarchy())
+      Analytics.handleEvent(ChunterEvents.MessageCreated, { ok: false, objectId, objectClass: object._class })
       Analytics.handleError(err)
       console.error(err)
```
Comment (haiodo) [inline plugins/chunter-resources/src/components/chat-message/ChatMessageInput.svelte:118]: do we still need them?

File: plugins/contact-resources/src/components/CreatePerson.svelte [hunk 3]
```diff
@@ -59,6 +60,7 @@
         provider: channel.provider
       })
     }
+    Analytics.handleEvent(ContactEvents.PersonCreated, { id: personId })
```
Comment (haiodo) [inline plugins/contact-resources/src/components/CreatePerson.svelte:63]: we need some kind of framework with contributions

Comment (kristina-fefelova) [reply plugins/contact-resources/src/components/CreatePerson.svelte:63]: Discussed in slack

File: plugins/contact-resources/src/components/CreateOrganization.svelte (hunk 1 above)
Comment (kristina-fefelova) [reply plugins/contact-resources/src/components/CreateOrganization.svelte:70]: Discussed in slack

File: plugins/chunter-resources/src/components/chat-message/ChatMessageInput.svelte (hunk 2 above)
Comment (kristina-fefelova) [reply plugins/chunter-resources/src/components/chat-message/ChatMessageInput.svelte:118]: I think yes, so we can handle errors in sentry

**Review by haiodo — APPROVED** [review]

**Pull request merged**